docker-entrypoint.sh

```diff
@@ -6,8 +6,6 @@ IFS=$'\n\t'
 
 export S3_ACL=${S3_ACL:-private}
 
-test $MOUNT_POINT
-rm -rf ${MOUNT_POINT}
```

Comment:
@andrius what's your rationale here? S3FS will fail to mount if the mountpoint isn't empty?